youtube_dl/extractor/antenna.py

```diff
@@ -0,0 +1,115 @@
+    _TESTS = [
+        {
+            'url': 'http://www.antenna.gr/webtv/watch?cid=jbq_kgua8_jw_a%3d&p=1',
+            'info_dict': {
+                'id': 'jbq_kgua8_jw_a%3d',
+                'ext': 'mp4',
+                'title': 'ANT1 News 19-09-2016 στις 19:00 \r\n',
+                'thumbnail': 're:^https?://.*\.jpg$',
+                'description': 'Μετά από αλλεπάλληλες αναβολές ξεκίνησε η δίκη για την τραγωδία της Marfin.',
+            },
+            'params': {
+                'skip_download': True,
+            },
+            'skip': 'Content not available outside Greece.'
+        },
+        {
+            'url': 'http://www.antenna.gr/webtv/watch?cid=%2fn_u4_x_n79i_d_i%3d',
+            'info_dict': {
+                'id': '%2fn_u4_x_n79i_d_i%3d',
+                'ext': 'mp4',
+                'title': 'Της Ελλάδος τα παιδιά (επεισ.38 - Ιφιγένεια εν Τατοϊω)',
+                'thumbnail': 're:^https?://.*\.jpg$',
+                'description': 'Τρείς σμηνίτες, η κυρία Μπoυμπoύ και o Σμήvαρχoς τoυ γραφείoυ Κάκαλoς, συνθέτουν μια '
+                               'από τις πιο αστείες συντροφιές της ελληνικής τηλεόρασης.',
+            },
+            'params': {
+                'skip_download': True,
+            },
+            'skip': 'Content not available outside Greece.'
+        }]
+
+    class MediaSelectionError(Exception):
+        def __init__(self, error_id):
+            self.id = error_id
+
+    def _raise_extractor_error(self, media_selection_error):
+        raise ExtractorError('{0} returned error: {1}'.format(self.IE_NAME,
+                                                              media_selection_error.id), expected=True)
+
+    def _extract_info_rss(self, video_id, webpage):
+        formats = []
+        jplayer_url = 'http://www.antenna.gr/templates/data/jplayer?d=m&cid='
+        desc_re = r'<description><!\[CDATA\[(.+?)\]\]></description>'
+        rss = self._download_webpage(jplayer_url + video_id, video_id)
+        video_url = self._search_regex(r'file="(.+?)"', rss, 'url')
+        if video_url == 'http://extranet.antenna.gr/flvsteaming/GR.flv':
+            raise AntennaIE.MediaSelectionError('Content not available outside Greece.')
+        formats.extend(self._extract_akamai_formats(video_url, video_id))
+        for video_format in formats:
+            if video_format.get('format_note') == 'Quality selection URL':
+                formats.remove(video_format)
+        if len(formats) == 0:
+            raise AntennaIE.MediaSelectionError('No formats available')
+        title = self._og_search_title(webpage).split(' | ')[-1] or self._search_regex(r'<title>(.+?)</title>', rss,
+                                                                                      'title')[9:-3]
+        thumbnail = self._og_search_thumbnail(webpage)
+        desc = self._og_search_description(webpage).split(' | ')[-1] or self._search_regex(desc_re, rss, 'description')
```

Comment:
In both extraction functions you extract these fields from `webpage` (as fallbacks in the second case) but you do it differently (split on ` | ` here and as is in the second case). What's the rationale?
`_search_regex` is fatal by default and will break extraction if it's not found for optional fields.